spring-integration-etcd/build.gradle

```diff
@@ -0,0 +1,190 @@
+description = 'Spring Integration Etcd Support'
+
+apply plugin: 'java'
+apply from: "${rootProject.projectDir}/publish-maven.gradle"
+apply plugin: 'eclipse'
+apply plugin: 'idea'
+apply plugin: 'jacoco'
+
+group = 'org.springframework.integration'
+
+buildscript {
+	repositories {
+		maven { url 'http://repo.spring.io/plugins-release' }
+        mavenCentral()
+	}
+	dependencies {
+		classpath 'io.spring.gradle:spring-io-plugin:0.0.4.RELEASE'
+	}
+}
+
+repositories {
+	if (version.endsWith('BUILD-SNAPSHOT') || project.hasProperty('platformVersion')) {
+		maven { url 'http://repo.spring.io/libs-snapshot' }
+	}
+	maven { url 'http://repo.spring.io/libs-milestone' }
+}
+
+if (project.hasProperty('platformVersion')) {
+	apply plugin: 'spring-io'
+
+	dependencyManagement {
+		springIoTestRuntime {
+			imports {
+				mavenBom "io.spring.platform:platform-bom:${platformVersion}"
+			}
+		}
+	}
+}
+
+sourceCompatibility = targetCompatibility = 1.7
+
+ext {
+	etcdVersion = '0.0.7'
+	slf4jVersion = '1.7.13'
+	springIntegrationVersion = '4.2.4.RELEASE'
```

Comment:
Please, upgrade to Spring Integration `5.0.8` (going to be `5.0.9` next week if that)